flags.yml

```diff
@@ -135,3 +135,10 @@
   default: false
   contact: Lyon Hill
   lifetime: temporary
+
+- name: Push Down Merged Filters
```

Comment:
Most of the other flags in this file have the `name` field very similar to the `key` field, can you follow that convention here?

Also, this rule is not really a "push down" since it's not actually moving any query execution into the storage layer, it's just enabling the merging of filters that are executed in Flux. Perhaps a better name would be `Merge Filters Rule` or something like that.